stripe/_http_client.py

```diff
@@ -1353,6 +1364,113 @@ async def close_async(self):
         await self._client_async.aclose()
 
 
+class AIOHTTPClient(HTTPClient):
+    name = "aiohttp"
+
+    def __init__(
+        self, timeout: Optional[Union[float, "AIOHTTPTimeout"]] = 80, **kwargs
+    ):
+        super(AIOHTTPClient, self).__init__(**kwargs)
+
+        if aiohttp is None:
+            raise ImportError(
+                "Unexpected: tried to initialize AIOHTTPClient but the aiohttp module is not present."
+            )
+
+        self.httpx = httpx
+
+        kwargs = {}
+        if self._verify_ssl_certs:
+            ssl_context = ssl.create_default_context(
+                capath=stripe.ca_bundle_path
+            )
+            kwargs["connector"] = TCPConnector(ssl=ssl_context)
+
+        self._session = aiohttp.ClientSession(**kwargs)
+        self._timeout = timeout
+
+    def sleep_async(self, secs):
+        return asyncio.sleep(secs)
+
+    def request(self) -> Tuple[bytes, int, Mapping[str, str]]:
```

Comment:
Do we want to implement sync fallback?

Reply:
I don't think so. Async fallback is mostly just so the default client can use it. And initializing a sync client like Requests and setting AIOHTTP as the async fallback achieves the same effect, except for the case where you want to use AIOHTTP for async requests but HTTPX for sync requests. If the user wants this, this should implement their own HTTPClient.